dns/dnscrypt-proxy/src/opnsense/mvc/app/models/OPNsense/Dnscryptproxy/General.xml

```diff
@@ -38,6 +38,10 @@
             <default>1</default>
             <Required>Y</Required>
         </doh_servers>
+        <odoh_servers type="BooleanField">
```

Comment:
This needs modification of the version at the beginning of this file.

Reply:
Done!